README.md

```diff
@@ -1,7 +1,8 @@
 # go-powershell
 
-![License](https://img.shields.io/github/license/gorillalabs/go-powershell.svg)
-[![GoDoc](https://godoc.org/github.com/gorillalabs/go-powershell?status.svg)](https://godoc.org/github.com/gorillalabs/go-powershell)
+# Original Version https://github.com/40a/go-powershell
```

Comment:
Let's say "forked from" rather than "original version" because it kinda looks like the original might once have been under gorillalabs